vhost/src/vhost_user/gpu_message.rs

```diff
@@ -342,7 +342,7 @@ unsafe impl ByteValued for VhostUserGpuDMABUFScanout2 {}
 impl VhostUserMsgValidator for VhostUserGpuDMABUFScanout2 {}
 
 /// The VhostUserGpuCursorPos from the QEMU vhost-user-gpu specification.
-#[derive(Copy, Clone, Debug, Default)]
+#[derive(Copy, Clone, Debug, Default, PartialEq)]
```

Comment:
Add this when introducing `VhostUserGpuCursorPos` and `VhostUserGpuCursorUpdate`